### PR TITLE
Setup dependabot and update the easiest 

### PR DESCRIPTION
**Pull request by mmorel-35**

### CODE REVIEW

**Review by phlax — COMMENTED** [review]
@mmorel-35 thanks for this - really appreciated

can we add the dependabot config in a single pr and then deal with any other changes and updates separately please - some of these can affect downstream and need to be tested

**Comment by phlax** [comment]
also worth mentioning - we have a `dependatool` that is hosted in this repo and used elsehwere - once we have this setup we should add it here - essentially it checks that the config matches what is present in repo and v-v

**Comment by mmorel-35** [comment]
Alright, let's start with #1277 then

File: gh-actions/appauth/.eslintrc.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
-    "prettier/@typescript-eslint",
```
Comment (phlax) [inline gh-actions/appauth/.eslintrc.json:7]: can you explain more why to remove this?

i think if we are going to make these kind of changes probs best do those next separately - but would like to understand better why to remove

Comment (mmorel-35) [reply gh-actions/appauth/.eslintrc.json:7]: It has been deprecated and included in "prettier".

**Comment by mmorel-35** [comment]
Replaced by smaller PR

**Pull request closed without merging**